### PR TITLE
Fix mixin problems.

### PR DESCRIPTION
**Pull request by JBYoshi**
- There are some explosive mixins that were written but not registered. I have registered them and removed a method that was implemented in the explosive mixin.
- There were two duplicate EnumArt mixins. I have removed the one in mixin.core.entity.hanging.art (which also wasn't registered).

### CODE REVIEW

File: src/main/java/org/spongepowered/common/mixin/core/entity/vehicle/minecart/MixinEntityMinecartTNT.java [hunk 1]
```diff
@@ -33,9 +32,4 @@
 @NonnullByDefault
 @Mixin(EntityMinecartTNT.class)
 public abstract class MixinEntityMinecartTNT extends MixinEntityMinecart implements MinecartTNT {
-
-    @Override
-    public FuseData getFuseData() {
```
Comment (Zidane) [inline src/main/java/org/spongepowered/common/mixin/core/entity/vehicle/minecart/MixinEntityMinecartTNT.java]: What was the reasoning for removing this?

Comment (Zidane) [inline src/main/java/org/spongepowered/common/mixin/core/entity/vehicle/minecart/MixinEntityMinecartTNT.java]: ignore comment above.

**Comment by Zidane** [comment]
Thanks for this @JBYoshi :).

**Comment by JBYoshi** [comment]
You're welcome.

**Pull request closed without merging**